site2/website/versioned_docs/version-2.10.x/reference-configuration.md

```diff
@@ -551,6 +551,7 @@ You can set the log level and configuration in the  [log4j2.yaml](https://github
 | dispatchThrottlingOnNonBacklogConsumerEnabled | Enable dispatch-throttling for both caught up consumers as well as consumers who have backlogs. | true |
 |dispatcherMaxReadBatchSize|The maximum number of entries to read from BookKeeper. By default, it is 100 entries.|100|
 |dispatcherMaxReadSizeBytes|The maximum size in bytes of entries to read from BookKeeper. By default, it is 5MB.|5242880|
+|dispatcherEntryFilterRescheduledMessageDelaySeconds|The max seconds a consumer will be paused because the current message cannot be consumed by a consumer. By default, it is 1.|1|
```

Comment:
this is in 2.10 version website, I don't think this is the correct place

Reply:
Already removed this line, because it is now automatically generated